signer/src/testing/context.rs

```diff
@@ -327,7 +330,9 @@ impl StacksInteract for WrappedMock<MockStacksInteract> {
     }
 
     fn nakamoto_start_height(&self) -> u64 {
-        tokio::runtime::Handle::current().block_on(async move {
+        let handle = tokio::runtime::Handle::current();
```

Comment:
You should be able to just `tokio::runtime::Handle::current().block_on(..)` I think?

Reply:
Previously it was like that, but when running the test it would fail with `Cannot start a runtime from within a runtime. This happens because a function (like `block_on`) attempted to block the current thread while the thread is being used to drive asynchronous tasks.`. I'm not sure if it's just because of the way the test run, though.

Reply:
Hm, weird :thinking: Yeah, must have something to do with the test runtime I would guess, not sure what `tokio::test` does under the covers..